Aztec/Classes/Formatters/Implementations/LinkFormatter.swift

```diff
@@ -2,13 +2,16 @@ import UIKit
 
 class LinkFormatter: StandardAttributeFormatter {
 
+    var target: String?
```

Comment:
I think it would be a bit cleaner if we changed this to:

```swift
private let target: String?
```

And changed the initializer to:

```swift
init(target: String? = nil) {
    self.target = target
    ...
```

Rationale: it's not really very likely that we will need to either read or write the target of a formatter again after instantiating it.